Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -375,7 +375,7 @@ protected override void ProcessRecord()
             catch (GoogleApiException ex) when (ex.HttpStatusCode == HttpStatusCode.Conflict)
             {
                 WriteError(new ErrorRecord(ex,  
-                    $"A dataset with the name {DatasetId} already exists.",
+                    $"A dataset with the name {DatasetId} already exists in this project.",
```

Comment:
Since you actually have the Project's name, I think a better error message would be:
`$"A dataset with the name '{DatasetId}' already exists in project '{Project}'."`